### DISCUSSION TITLE
How to show total count of related record on Manage Related Record tab badge ?

### DISCUSSION THREAD

**Discussion by AsmitNepali** [filamentphp/filament#13054] [Help]
### Package

Panel builder

### Package Version

v3

### How can we help you?

Count the related record and show ton the navigation badge like this.
<img width="141" alt="Screenshot 2024-06-01 at 19 12 03" src="https://github.com/filamentphp/filament/assets/33512152/72b5406e-099b-4f26-bf45-251d276e769c">

**Comment by rezadonny** [comment]
i have the same question.. please help. Thanks in advance.
